spec/instant.html

```diff
@@ -576,7 +576,9 @@ <h1>DifferenceInstant ( _ns1_, _ns2_, _roundingIncrement_, _smallestUnit_, _roun
       <emu-alg>
         1. Assert: Type(_ns1_) is BigInt.
         1. Assert: Type(_ns2_) is BigInt.
-        1. Return ! RoundTemporalInstant(_ns2_ - _ns1_, _roundingIncrement_, _smallestUnit_, _roundingMode_).
+        1. Let _roundResult_ be ? RoundDuration(0, 0, 0, 0, 0, 0, 0, 0, 0, _ns2_ - _ns1_, _roundingIncrement_, _smallestUnit_, _roundingMode_).
+        1. Let _nanoseconds_ be ? BalanceDuration(_roundResult_.[[Days]], _roundResult_.[[Hours]], _roundResult_.[[Minutes]], _roundResult_.[[Seconds]], _roundResult_.[[Milliseconds]], _roundResult_.[[Microseconds]], _roundResult_.[[Nanoseconds]], *"nanosecond"*).[[Nanoseconds]].
```

Comment:
smallestUnit can only be hours and below, so I think we could do "Assert: _roundResult_.[[Days]]_ = 0" and fill in 0 for the days argument here.